homeassistant/helpers/helper_config_entry_flow.py

```diff
@@ -337,3 +366,22 @@ def wrapped_entity_config_entry_title(
     if state:
         return state.name or object_id
     return object_id
+
+
+@callback
+def exclude_own_entities(
+    handler: HelperConfigFlowHandler | HelperOptionsFlowHandler,
```

Comment:
Since this method is only used with options schema, why do we even allow passing config flow handlers ?

Reply:
Fixed in cc8a589